chatbot/ai-lab.yaml

```diff
@@ -5,8 +5,8 @@ application:
   description: This is a Streamlit chat demo application. 
   containers:
     - name: llamacpp-server
-      contextdir: ../playground
-      containerfile: Containerfile
+      contextdir: ../model_servers/llamacpp_python
+      containerfile: ../model_servers/llamacpp_python/base/Containerfile
```

Comment:
I didn't notice before but i think that the containerfile should be `containerfile: ./base/Containerfile`

Reply:
This is related to all ai-lab files

Reply:
Ok, was was actually not sure. It will start from the contextDir?

Reply:
yes